manager/manager.go

```diff
@@ -71,7 +71,7 @@ type Config struct {
 
 	// RemoteAPI is a listening address for serving the remote API, and
 	// an optional advertise address.
-	RemoteAPI RemoteAddrs
+	RemoteAPI *RemoteAddrs
```

Comment:
What you're suggesting should work fine. I felt that since `RemoteAddrs` is a struct, it might be confusing to check one particular field to see if the struct is initialized.